host-controller/src/main/java/org/jboss/as/host/controller/operations/NativeManagementAddHandler.java

```diff
@@ -81,6 +85,7 @@ protected void installServices(OperationContext context, NativeInterfaceCommonPo
         NativeManagementServices.installManagementWorkerService(serviceTarget, context.getServiceRegistry(false));
         ManagementRemotingServices.installDomainConnectorServices(serviceTarget, ManagementRemotingServices.MANAGEMENT_ENDPOINT,
                 nativeManagementInterfaceBinding, hostControllerInfo.getNativeManagementPort(), securityRealm, options);
+        return Arrays.asList(ManagementRemotingServices.MANAGEMENT_ENDPOINT, UndertowHttpManagementService.SERVICE_NAME);
```

Comment:
The question here is I don't think UndertowHttpManagementService.SERVICE_NAME is related to native management.

For native I believe you want the service added by ManagementRemotingServices.installDomainConnectorServices, which is RemotingServices.serverServiceName(connectorName). That's the service that opens a socket for native requests. It depends on the various others like the endpoint so it won't be UP unless the others are.